airflow/dag_processing/processor.py

```diff
@@ -549,6 +549,17 @@ def update_import_errors(session: Session, dagbag: DagBag) -> None:
                     synchronize_session='fetch',
                 )
             else:
+                emails: Set[str] = set(conf.get(section='email', key='dag_import_failure_email_to').split())
+                if emails and conf.getboolean(
+                    section='email', key='email_on_dag_import_failure', fallback=True
+                ):
+                    try:
+                        email_content = f"""File {filename} has failed to import due to:\n
+                        <pre><code>{stacktrace}<code></pre>
```

Comment:
I'm not sure if the emails are plain text or html, though I assume plain text. So this doesn't seem right.

Reply:
The emails show up and look as expected, it does seem to be HTML